### ISSUE TITLE
Getting Errno::EADDRINUSE error="Address already in use - bind(2) for 0.0.0.0:24220"

### ISSUE THREAD

**Issue by dcvtruong** [fluent/fluentd#2586]
Check [CONTRIBUTING guideline](https://github.com/fluent/fluentd/blob/master/CONTRIBUTING.md) first and here is the list to help us investigate the problem.

**Describe the bug**
Fluentd startup error 'Address already in use - bind(2) for 0.0.0.0:24220' message repeatedly.

**To Reproduce**
Use the config file listed below.

**Expected behavior**
Should not be getting the Address already in use... error as I did not add any new worker

**Your Environment**

- Fluentd 1.6.3
- Operating system: red hat 7.6
- Kernel version: 3.10.0-957.21.3.el7.x86_64

If you hit the problem with older fluentd version, try latest version first.

**Your Configuration**

```
      2019-08-26 23:48:25 +0000 [info]: parsing config file is succeeded path="/etc/fluent/fluent.conf"
      2019-08-26 23:48:26 +0000 [info]: using configuration file: <ROOT>
        <match fluent.**>
          @type null
        </match>
        <source>
          @id fluentd-containers.log
          @type tail
          path "/var/log/containers/*.log"
          pos_file "/var/log/containers.log.pos"
          tag "raw.kubernetes.*"
          read_from_head true
          <parse>
            @type "multi_format"
            <pattern>
              format json
              time_key "time"
              time_format "%Y-%m-%dT%H:%M:%S.%NZ"
              time_type string
            </pattern>
            <pattern>
              format /^(?<time>.+) (?<stream>stdout|stderr) [^ ]* (?<log>.*)$/
              time_format "%Y-%m-%dT%H:%M:%S.%N%:z"
              expression ^(?<time>.+) (?<stream>stdout|stderr) [^ ]* (?<log>.*)$
              ignorecase false
              multiline false
            </pattern>
          </parse>
        </source>
        <match raw.kubernetes.**>
          @id raw.kubernetes
          @type detect_exceptions
          remove_tag_prefix "raw"
          message "log"
          stream "stream"
          multiline_flush_interval 5
          max_bytes 500000
          max_lines 1000
        </match>
        <filter **>
          @id filter_concat
          @type concat
          key "message"
          multiline_end_regexp "/\\n$/"
          separator ""
        </filter>
        <filter kubernetes.**>
          @id filter_kubernetes_metadata
          @type kubernetes_metadata
        </filter>
        <filter kubernetes.**>
          @id filter_parser
          @type parser
          key_name "log"
          reserve_data true
          remove_key_name_field true
          <parse>
            @type "multi_format"
            <pattern>
              format json
            </pattern>
            <pattern>
              format none
            </pattern>
          </parse>
        </filter>
        <source>
          @id forward
          @type forward
        </source>
        <source>
          @id prometheus
          @type prometheus
        </source>
        <source>
          @id monitor_agent
          @type monitor_agent
        </source>
        <source>
          @id prometheus_monitor
          @type prometheus_monitor
          <labels>
            host ${hostname}
          </labels>
        </source>
        <source>
          @id prometheus_output_monitor
          @type prometheus_output_monitor
          <labels>
            host ${hostname}
          </labels>
        </source>
        <source>
          @id prometheus_tail_monitor
          @type prometheus_tail_monitor
          <labels>
            host ${hostname}
          </labels>
        </source>
        <match **>
          @id elasticsearch
          @type elasticsearch
          @log_level "info"
          include_tag_key true
          type_name "_doc"
          host "elastic.ew.oc.001.private.theagilehub.net"
          port 30998
          scheme http
          ssl_version TLSv1_2
          ssl_verify true
          user ""
          password xxxxxx
          logstash_format true
          logstash_prefix "logstash"
          reconnect_on_error true
          <buffer>
            @type "file"
            path "/var/log/fluentd-buffers/kubernetes.system.buffer"
            flush_mode interval
            retry_type exponential_backoff
            flush_thread_count 2
            flush_interval 5s
            retry_forever 
            retry_max_interval 30
            chunk_limit_size 2M
            queue_limit_length 8
            overflow_action block
          </buffer>
        </match>
        <system>
          root_dir "/tmp/fluentd-buffers/"
        </system>
        <source>
          @id minion
          @type tail
          format /^(?<time>[^ ]* [^ ,]*)[^\[]*\[[^\]]*\]\[(?<severity>[^ \]]*) *\] (?<message>.*)$/
          time_format %Y-%m-%d %H:%M:%S
          path "/var/log/salt/minion"
          pos_file "/var/log/salt.pos"
          tag "salt"
          <parse>
            time_format %Y-%m-%d %H:%M:%S
            @type regexp
            expression ^(?<time>[^ ]* [^ ,]*)[^\[]*\[[^\]]*\]\[(?<severity>[^ \]]*) *\] (?<message>.*)$
          </parse>
        </source>
        <source>
          @id startupscript.log
          @type tail
          format syslog
          path "/var/log/startupscript.log"
          pos_file "/var/log/startupscript.log.pos"
          tag "startupscript"
          <parse>
            @type syslog
          </parse>
        </source>
        <source>
          @id docker.log
          @type tail
          format /^time="(?<time>[^)]*)" level=(?<severity>[^ ]*) msg="(?<message>[^"]*)"( err="(?<error>[^"]*)")?( statusCode=($<status_code>\d+))?/
          path "/var/log/docker.log"
          pos_file "/var/log/docker.log.pos"
          tag "docker"
          <parse>
            @type regexp
            expression ^time="(?<time>[^)]*)" level=(?<severity>[^ ]*) msg="(?<message>[^"]*)"( err="(?<error>[^"]*)")?( statusCode=($<status_code>\d+))?
          </parse>
        </source>
        <source>
          @id etcd.log
          @type tail
          format none
          path "/var/log/etcd.log"
          pos_file "/var/log/etcd.log.pos"
          tag "etcd"
          <parse>
            @type none
          </parse>
        </source>
        <source>
          @id kubelet.log
          @type tail
          format multiline
          multiline_flush_interval 5s
          format_firstline /^\w\d{4}/
          format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
          time_format %m%d %H:%M:%S.%N
          path "/var/log/kubelet.log"
          pos_file "/var/log/kubelet.log.pos"
          tag "kubelet"
          <parse>
            time_format %m%d %H:%M:%S.%N
            format_firstline /^\w\d{4}/
            @type multiline
            format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
          </parse>
        </source>
        <source>
          @id kube-proxy.log
          @type tail
          format multiline
          multiline_flush_interval 5s
          format_firstline /^\w\d{4}/
          format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
          time_format %m%d %H:%M:%S.%N
          path "/var/log/kube-proxy.log"
          pos_file "/var/log/kube-proxy.log.pos"
          tag "kube-proxy"
          <parse>
            time_format %m%d %H:%M:%S.%N
            format_firstline /^\w\d{4}/
            @type multiline
            format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
          </parse>
        </source>
        <source>
          @id kube-apiserver.log
          @type tail
          format multiline
          multiline_flush_interval 5s
          format_firstline /^\w\d{4}/
          format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
          time_format %m%d %H:%M:%S.%N
          path "/var/log/kube-apiserver.log"
          pos_file "/var/log/kube-apiserver.log.pos"
          tag "kube-apiserver"
          <parse>
            time_format %m%d %H:%M:%S.%N
            format_firstline /^\w\d{4}/
            @type multiline
            format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
          </parse>
        </source>
        <source>
          @id kube-controller-manager.log
          @type tail
          format multiline
          multiline_flush_interval 5s
          format_firstline /^\w\d{4}/
          format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
          time_format %m%d %H:%M:%S.%N
          path "/var/log/kube-controller-manager.log"
          pos_file "/var/log/kube-controller-manager.log.pos"
          tag "kube-controller-manager"
          <parse>
            time_format %m%d %H:%M:%S.%N
            format_firstline /^\w\d{4}/
            @type multiline
            format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
          </parse>
        </source>
        <source>
          @id kube-scheduler.log
          @type tail
          format multiline
          multiline_flush_interval 5s
          format_firstline /^\w\d{4}/
          format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
          time_format %m%d %H:%M:%S.%N
          path "/var/log/kube-scheduler.log"
          pos_file "/var/log/kube-scheduler.log.pos"
          tag "kube-scheduler"
          <parse>
            time_format %m%d %H:%M:%S.%N
            format_firstline /^\w\d{4}/
            @type multiline
            format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
          </parse>
        </source>
        <source>
          @id glbc.log
          @type tail
          format multiline
          multiline_flush_interval 5s
          format_firstline /^\w\d{4}/
          format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
          time_format %m%d %H:%M:%S.%N
          path "/var/log/glbc.log"
          pos_file "/var/log/glbc.log.pos"
          tag "glbc"
          <parse>
            time_format %m%d %H:%M:%S.%N
            format_firstline /^\w\d{4}/
            @type multiline
            format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
          </parse>
        </source>
        <source>
          @id cluster-autoscaler.log
          @type tail
          format multiline
          multiline_flush_interval 5s
          format_firstline /^\w\d{4}/
          format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
          time_format %m%d %H:%M:%S.%N
          path "/var/log/cluster-autoscaler.log"
          pos_file "/var/log/cluster-autoscaler.log.pos"
          tag "cluster-autoscaler"
          <parse>
            time_format %m%d %H:%M:%S.%N
            format_firstline /^\w\d{4}/
            @type multiline
            format1 /^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<message>.*)/
          </parse>
        </source>
        <source>
          @id journald-docker
          @type systemd
          matches [{"_SYSTEMD_UNIT":"docker.service"}]
          read_from_head true
          tag "docker"
          <storage>
            @type "local"
            persistent true
            path "/var/log/journald-docker.pos"
          </storage>
        </source>
        <source>
          @id journald-container-runtime
          @type systemd
          matches [{"_SYSTEMD_UNIT":"{{ fluentd_container_runtime_service }}.service"}]
          read_from_head true
          tag "container-runtime"
          <storage>
            @type "local"
            persistent true
            path "/var/log/journald-container-runtime.pos"
          </storage>
        </source>
        <source>
          @id journald-kubelet
          @type systemd
          matches [{"_SYSTEMD_UNIT":"kubelet.service"}]
          read_from_head true
          tag "kubelet"
          <storage>
            @type "local"
            persistent true
            path "/var/log/journald-kubelet.pos"
          </storage>
        </source>
        <source>
          @id journald-node-problem-detector
          @type systemd
          matches [{"_SYSTEMD_UNIT":"node-problem-detector.service"}]
          read_from_head true
          tag "node-problem-detector"
          <storage>
            @type "local"
            persistent true
            path "/var/log/journald-node-problem-detector.pos"
          </storage>
        </source>
        <source>
          @id kernel
          @type systemd
          matches [{"_TRANSPORT":"kernel"}]
          read_from_head true
          tag "kernel"
          <storage>
            @type "local"
            persistent true
            path "/var/log/kernel.pos"
          </storage>
          <entry>
            fields_strip_underscores true
            fields_lowercase true
          </entry>
        </source>
      </ROOT>
      2019-08-26 23:48:26 +0000 [info]: starting fluentd-1.6.3 pid=60 ruby="2.3.3"
      2019-08-26 23:48:26 +0000 [info]: spawn command to main:  cmdline=["/usr/bin/ruby2.3", "-Eascii-8bit:ascii-8bit", "/usr/local/bin/fluentd", "--under-supervisor"]
      2019-08-26 23:48:27 +0000 [info]: gem 'fluent-plugin-concat' version '2.4.0'
      2019-08-26 23:48:27 +0000 [info]: gem 'fluent-plugin-detect-exceptions' version '0.0.12'
      2019-08-26 23:48:27 +0000 [info]: gem 'fluent-plugin-elasticsearch' version '3.5.4'
      2019-08-26 23:48:27 +0000 [info]: gem 'fluent-plugin-kubernetes_metadata_filter' version '2.2.0'
      2019-08-26 23:48:27 +0000 [info]: gem 'fluent-plugin-multi-format-parser' version '1.0.0'
      2019-08-26 23:48:27 +0000 [info]: gem 'fluent-plugin-prometheus' version '1.4.0'
      2019-08-26 23:48:27 +0000 [info]: gem 'fluent-plugin-systemd' version '1.0.2'
      2019-08-26 23:48:27 +0000 [info]: gem 'fluentd' version '1.6.3'
      2019-08-26 23:48:27 +0000 [info]: adding match pattern="fluent.**" type="null"
      2019-08-26 23:48:27 +0000 [info]: adding match pattern="raw.kubernetes.**" type="detect_exceptions"
      2019-08-26 23:48:27 +0000 [info]: adding filter pattern="**" type="concat"
      2019-08-26 23:48:27 +0000 [info]: adding filter pattern="kubernetes.**" type="kubernetes_metadata"
      2019-08-26 23:48:27 +0000 [info]: adding filter pattern="kubernetes.**" type="parser"
      2019-08-26 23:48:27 +0000 [info]: adding match pattern="**" type="elasticsearch"
      2019-08-26 23:48:28 +0000 [info]: adding source type="tail"
      2019-08-26 23:48:28 +0000 [info]: adding source type="forward"
      2019-08-26 23:48:28 +0000 [info]: adding source type="prometheus"
      2019-08-26 23:48:28 +0000 [info]: adding source type="monitor_agent"
      2019-08-26 23:48:28 +0000 [info]: adding source type="prometheus_monitor"
      2019-08-26 23:48:28 +0000 [info]: adding source type="prometheus_output_monitor"
      2019-08-26 23:48:28 +0000 [info]: adding source type="prometheus_tail_monitor"
      2019-08-26 23:48:28 +0000 [info]: adding source type="tail"
      2019-08-26 23:48:28 +0000 [info]: adding source type="tail"
      2019-08-26 23:48:28 +0000 [info]: adding source type="tail"
      2019-08-26 23:48:28 +0000 [info]: adding source type="tail"
      2019-08-26 23:48:28 +0000 [info]: adding source type="tail"
      2019-08-26 23:48:28 +0000 [info]: adding source type="tail"
      2019-08-26 23:48:28 +0000 [info]: adding source type="tail"
      2019-08-26 23:48:28 +0000 [info]: adding source type="tail"
      2019-08-26 23:48:28 +0000 [info]: adding source type="tail"
      2019-08-26 23:48:28 +0000 [info]: adding source type="tail"
      2019-08-26 23:48:28 +0000 [info]: adding source type="tail"
      2019-08-26 23:48:28 +0000 [info]: adding source type="systemd"
      2019-08-26 23:48:28 +0000 [info]: adding source type="systemd"
      2019-08-26 23:48:28 +0000 [info]: adding source type="systemd"
      2019-08-26 23:48:28 +0000 [info]: adding source type="systemd"
      2019-08-26 23:48:28 +0000 [info]: adding source type="systemd"
      2019-08-26 23:48:28 +0000 [info]: #0 starting fluentd worker pid=69 ppid=60 worker=0
```

**Log**

```
      2019-08-26 23:48:28 +0000 [error]: #0 unexpected error error_class=Errno::EADDRINUSE error="Address already in use - bind(2) for 0.0.0.0:24220"
        2019-08-26 23:48:28 +0000 [error]: #0 /usr/lib/ruby/2.3.0/socket.rb:205:in `bind'
        2019-08-26 23:48:28 +0000 [error]: #0 /usr/lib/ruby/2.3.0/socket.rb:205:in `listen'
        2019-08-26 23:48:28 +0000 [error]: #0 /usr/lib/ruby/2.3.0/socket.rb:759:in `block in tcp_server_sockets'
        2019-08-26 23:48:28 +0000 [error]: #0 /usr/lib/ruby/2.3.0/socket.rb:231:in `each'
        2019-08-26 23:48:28 +0000 [error]: #0 /usr/lib/ruby/2.3.0/socket.rb:231:in `foreach'
        2019-08-26 23:48:28 +0000 [error]: #0 /usr/lib/ruby/2.3.0/socket.rb:757:in `tcp_server_sockets'
        2019-08-26 23:48:28 +0000 [error]: #0 /usr/lib/ruby/2.3.0/webrick/utils.rb:65:in `create_listeners'
        2019-08-26 23:48:28 +0000 [error]: #0 /usr/lib/ruby/2.3.0/webrick/server.rb:134:in `listen'
        2019-08-26 23:48:28 +0000 [error]: #0 /usr/lib/ruby/2.3.0/webrick/server.rb:115:in `initialize'
        2019-08-26 23:48:28 +0000 [error]: #0 /usr/lib/ruby/2.3.0/webrick/httpserver.rb:47:in `initialize'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/lib/fluent/plugin_helper/http_server/compat/server.rb:31:in `new'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/lib/fluent/plugin_helper/http_server/compat/server.rb:31:in `initialize'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/lib/fluent/plugin_helper/http_server.rb:47:in `new'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/lib/fluent/plugin_helper/http_server.rb:47:in `create_http_server'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/lib/fluent/plugin/in_monitor_agent.rb:211:in `start'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/lib/fluent/root_agent.rb:203:in `block in start'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/lib/fluent/root_agent.rb:192:in `block (2 levels) in lifecycle'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/lib/fluent/root_agent.rb:191:in `each'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/lib/fluent/root_agent.rb:191:in `block in lifecycle'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/lib/fluent/root_agent.rb:178:in `each'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/lib/fluent/root_agent.rb:178:in `lifecycle'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/lib/fluent/root_agent.rb:202:in `start'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/lib/fluent/engine.rb:274:in `start'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/lib/fluent/engine.rb:219:in `run'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/lib/fluent/supervisor.rb:808:in `run_engine'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/lib/fluent/supervisor.rb:551:in `block in run_worker'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/lib/fluent/supervisor.rb:733:in `main_process'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/lib/fluent/supervisor.rb:546:in `run_worker'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/lib/fluent/command/fluentd.rb:320:in `<top (required)>'
        2019-08-26 23:48:28 +0000 [error]: #0 /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        2019-08-26 23:48:28 +0000 [error]: #0 /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        2019-08-26 23:48:28 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.6.3/bin/fluentd:8:in `<top (required)>'
        2019-08-26 23:48:28 +0000 [error]: #0 /usr/local/bin/fluentd:22:in `load'
        2019-08-26 23:48:28 +0000 [error]: #0 /usr/local/bin/fluentd:22:in `<main>'
      2019-08-26 23:51:39 +0000 [error]: #0 unexpected error error_class=Errno::EADDRINUSE error="Address already in use - bind(2) for 0.0.0.0:24220"
        2019-08-26 23:51:39 +0000 [error]: #0 suppressed same stacktrace
      2019-08-26 23:51:40 +0000 [info]: Worker 0 finished unexpectedly with status 1
      2019-08-26 23:51:40 +0000 [info]: gem 'fluent-plugin-concat' version '2.4.0'
      2019-08-26 23:51:40 +0000 [info]: gem 'fluent-plugin-detect-exceptions' version '0.0.12'
      2019-08-26 23:51:40 +0000 [info]: gem 'fluent-plugin-elasticsearch' version '3.5.4'
      2019-08-26 23:51:40 +0000 [info]: gem 'fluent-plugin-kubernetes_metadata_filter' version '2.2.0'
      2019-08-26 23:51:40 +0000 [info]: gem 'fluent-plugin-multi-format-parser' version '1.0.0'
      2019-08-26 23:51:40 +0000 [info]: gem 'fluent-plugin-prometheus' version '1.4.0'
      2019-08-26 23:51:40 +0000 [info]: gem 'fluent-plugin-systemd' version '1.0.2'
      2019-08-26 23:51:40 +0000 [info]: gem 'fluentd' version '1.6.3'
      2019-08-26 23:51:40 +0000 [info]: adding match pattern="fluent.**" type="null"
      2019-08-26 23:51:40 +0000 [info]: adding match pattern="raw.kubernetes.**" type="detect_exceptions"
      2019-08-26 23:51:40 +0000 [info]: adding filter pattern="**" type="concat"
      2019-08-26 23:51:40 +0000 [info]: adding filter pattern="kubernetes.**" type="kubernetes_metadata"
      2019-08-26 23:51:41 +0000 [info]: adding filter pattern="kubernetes.**" type="parser"
      2019-08-26 23:51:41 +0000 [info]: adding match pattern="**" type="elasticsearch"
      2019-08-26 23:51:41 +0000 [info]: adding source type="tail"
      2019-08-26 23:51:41 +0000 [info]: adding source type="forward"
      2019-08-26 23:51:41 +0000 [info]: adding source type="prometheus"
      2019-08-26 23:51:41 +0000 [info]: adding source type="monitor_agent"
      2019-08-26 23:51:41 +0000 [info]: adding source type="prometheus_monitor"
      2019-08-26 23:51:41 +0000 [info]: adding source type="prometheus_output_monitor"
      2019-08-26 23:51:41 +0000 [info]: adding source type="prometheus_tail_monitor"
      2019-08-26 23:51:41 +0000 [info]: adding source type="tail"
      2019-08-26 23:51:41 +0000 [info]: adding source type="tail"
      2019-08-26 23:51:41 +0000 [info]: adding source type="tail"
      2019-08-26 23:51:41 +0000 [info]: adding source type="tail"
      2019-08-26 23:51:41 +0000 [info]: adding source type="tail"
      2019-08-26 23:51:41 +0000 [info]: adding source type="tail"
      2019-08-26 23:51:41 +0000 [info]: adding source type="tail"
      2019-08-26 23:51:41 +0000 [info]: adding source type="tail"
      2019-08-26 23:51:41 +0000 [info]: adding source type="tail"
      2019-08-26 23:51:41 +0000 [info]: adding source type="tail"
      2019-08-26 23:51:41 +0000 [info]: adding source type="tail"
      2019-08-26 23:51:41 +0000 [info]: adding source type="systemd"
      2019-08-26 23:51:41 +0000 [info]: adding source type="systemd"
      2019-08-26 23:51:41 +0000 [info]: adding source type="systemd"
      2019-08-26 23:51:41 +0000 [info]: adding source type="systemd"
      2019-08-26 23:51:41 +0000 [info]: adding source type="systemd"
      2019-08-26 23:51:41 +0000 [info]: #0 starting fluentd worker pid=2949 ppid=60 worker=0
```

**Additional context**

**Comment by repeatedly** [comment]
You need to idenfity what the process uses its port before fluentd start.
Please continue the debug on mailing list.
If this is the bug of fluentd core, reopen the issue.

**Comment by Watson1978** [comment]
＠iDevPro Your issue is releated to https://github.com/fluent/fluentd/discussions/5100 https://github.com/fluent/fluentd/discussions/5103

We have a plan to fix the problem at Fluentd 1.19.2 or later.

So far, you can avoid the problem with:

```
<system>
  config_include_dir ""
</system>
```

Ref. https://github.com/fluent/fluentd/discussions/5100#discussioncomment-14422239
